src/Components/Template.tsx

```diff
@@ -0,0 +1,196 @@
+
+import oceanMusic from "../Audio/ocean.mp3";
+import neutral from "../Audio/summer.mp3";
+import extreme from "../Audio/extreme.mp3";
+import jazzy from "../Audio/jazzy.mp3";
+import gentle from "../Audio/gentle.mp3";
+import uplifting from "../Audio/uplifting.mp3";
+
+// duration is the length of each media displayed in video
+// if there are too many photos, then mb dont show all? (for now)
+// length is the length of the final video
+// durations and lengths in seconds
+//
+
+export type slotEl = {
+    length: "short" | "medium" | "long",
+    slot:number[],
+    slotLength: number,
+}
+
+export type templateEl = {
+    title: string,
+    musicTrack: string,
+    musicDuration: number,
+    musicName: string,
+    musicOptions: string[],
+    short: slotEl,
+    medium: slotEl,
+    long: slotEl,
+}
+
+
+export const templates:{[name: string]: templateEl} = {
+    calm: 
+    {
+        title: "calm",
+        musicTrack: oceanMusic,
+        musicDuration: 101,
+        musicName: "ocean sound",
+        musicOptions: [],
+        short: {
+            length: "short",
+            slot:[14,14,12],
+            slotLength: 40,
```

Comment:
The name of this value isn't clear, but after adding it up I understand that it's the sum of all the slots (14 + 14 + 12). Maybe `totalDuration`